pallets/gear/src/weights.rs

```diff
@@ -16,159 +16,1753 @@
-	fn submit_program(c: u32, p: u32) -> Weight;
-	fn send_message(p: u32) -> Weight;
-	fn send_reply(p: u32) -> Weight;
+	fn submit_code(c: u32, ) -> Weight;
+	fn submit_program(c: u32, p: u32, ) -> Weight;
+	fn send_message(p: u32, ) -> Weight;
+	fn send_reply(p: u32, ) -> Weight;
+	fn initial_allocation(q: u32, ) -> Weight;
+	fn alloc_in_handle(q: u32, ) -> Weight;
+	fn instr_i64const(r: u32, ) -> Weight;
+	fn instr_i64load(r: u32, ) -> Weight;
+	fn instr_i64store(r: u32, ) -> Weight;
+	fn instr_select(r: u32, ) -> Weight;
+	fn instr_if(r: u32, ) -> Weight;
+	fn instr_br(r: u32, ) -> Weight;
+	fn instr_br_if(r: u32, ) -> Weight;
+	fn instr_br_table(r: u32, ) -> Weight;
+	fn instr_br_table_per_entry(e: u32, ) -> Weight;
+	fn instr_call(r: u32, ) -> Weight;
+	fn instr_call_indirect(r: u32, ) -> Weight;
+	fn instr_call_indirect_per_param(p: u32, ) -> Weight;
+	fn instr_local_get(r: u32, ) -> Weight;
+	fn instr_local_set(r: u32, ) -> Weight;
+	fn instr_local_tee(r: u32, ) -> Weight;
+	fn instr_global_get(r: u32, ) -> Weight;
+	fn instr_global_set(r: u32, ) -> Weight;
+	fn instr_memory_current(r: u32, ) -> Weight;
+	fn instr_memory_grow(r: u32, ) -> Weight;
+	fn instr_i64clz(r: u32, ) -> Weight;
+	fn instr_i64ctz(r: u32, ) -> Weight;
+	fn instr_i64popcnt(r: u32, ) -> Weight;
+	fn instr_i64eqz(r: u32, ) -> Weight;
+	fn instr_i64extendsi32(r: u32, ) -> Weight;
+	fn instr_i64extendui32(r: u32, ) -> Weight;
+	fn instr_i32wrapi64(r: u32, ) -> Weight;
+	fn instr_i64eq(r: u32, ) -> Weight;
+	fn instr_i64ne(r: u32, ) -> Weight;
+	fn instr_i64lts(r: u32, ) -> Weight;
+	fn instr_i64ltu(r: u32, ) -> Weight;
+	fn instr_i64gts(r: u32, ) -> Weight;
+	fn instr_i64gtu(r: u32, ) -> Weight;
+	fn instr_i64les(r: u32, ) -> Weight;
+	fn instr_i64leu(r: u32, ) -> Weight;
+	fn instr_i64ges(r: u32, ) -> Weight;
+	fn instr_i64geu(r: u32, ) -> Weight;
+	fn instr_i64add(r: u32, ) -> Weight;
+	fn instr_i64sub(r: u32, ) -> Weight;
+	fn instr_i64mul(r: u32, ) -> Weight;
+	fn instr_i64divs(r: u32, ) -> Weight;
+	fn instr_i64divu(r: u32, ) -> Weight;
+	fn instr_i64rems(r: u32, ) -> Weight;
+	fn instr_i64remu(r: u32, ) -> Weight;
+	fn instr_i64and(r: u32, ) -> Weight;
+	fn instr_i64or(r: u32, ) -> Weight;
+	fn instr_i64xor(r: u32, ) -> Weight;
+	fn instr_i64shl(r: u32, ) -> Weight;
+	fn instr_i64shrs(r: u32, ) -> Weight;
+	fn instr_i64shru(r: u32, ) -> Weight;
+	fn instr_i64rotl(r: u32, ) -> Weight;
+	fn instr_i64rotr(r: u32, ) -> Weight;
 }
 
-/// Weight functions for `pallet_gear`.
+/// Weights for pallet_gear using the Gear node and recommended hardware.
 pub struct GearWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for GearWeight<T> {
 	// Storage: unknown [0x673a3a636f64653a3ac9cc4c0689648a67c955143538b059c3522b99d4b3f4d4] (r:1 w:1)
 	// Storage: unknown [0x673a3a636f64653a3a6d657461646174613a3ac9cc4c0689648a67c955143538] (r:0 w:1)
+	// Storage: unknown [0x673a3a6f726967636f64653a3ac9cc4c0689648a67c955143538b059c3522b99] (r:0 w:1)
```

Comment:
what is this spam?

Reply:
> what is this spam?

Storage interactions are placed in comments by default, I'll remove this before merging.

Reply:
This is autogenerated file so it seems normal to leave those comments